lib/common_test/src/ct_ssh.erl

```diff
@@ -405,15 +589,25 @@ send_and_receive(SSH, ChannelId, Type, Data, End) when is_function(End) ->
     send_and_receive(SSH, ChannelId, Type, Data, End, ?DEFAULT_TIMEOUT).
 
 -doc """
-send_and_receive(SSH, ChannelId, Type, Data, End, Timeout) -> {ok, Data} |
-{error, Reason}
+send_and_receive(SSH, ChannelId, Type, Data, End, Timeout) -> {ok, ReceivedData}
```

Comment:
Is it ok to just change documentation regarding {timeout, ReceivedData}?

Reply:
I am not sure I follow this and below.  Is it the case that the code never returns {error, Reason} but sometimes {timeout, ReceivedData} ?  If the code documents something that never happens I think it ok to say it is a documentation bug and remove that . If there is a question about changing the variable name in the spec to be more descriptive  that is no problem either. If the question is something else please explain.

Reply:
Sorry it's not clear from the part of code that is displayed. All cases in ct_ssh.erl are as follows.
Documentation states that it functions can return:
`{ok, Data} | {error, Reason}`
But in reality they can return:
`{ok, Data} | {timeout, Data} | {error, Reason}`

I think documentation should be fixed in ct_ssh.erl cases.

Reply:
Yes I think it is ok to update the documentation to reflect how the function has always worked.

Reply:
Fixed in documentation.